main.js

```diff
@@ -19,15 +19,19 @@ function Hoverable() {
 	var win = window;
 
 	function init() {
-		win.document.body.setAttribute('data-o-hoverable--js', '');
+		win.document.documentElement.setAttribute('data-o-hoverable--js', '');
 		touchSupport();
 	}
 
+	function updateClasses(action) {
```

Comment:
This looks like an over refactor, no?

Also, is it really important to add/remove the class on the body at all? What negative side effects can it have to keep it? This module is so rarely used in products it's very probably that you can get away with not removing it without breaking functionality.